### PR TITLE
feat(onboarding): Onboarding task setup integrations

### PR DESCRIPTION
**Pull request by PixelDust22**
This adds a new onboarding task to be used later for taking people back to onboarding integration select.

### CODE REVIEW

File: src/sentry/receivers/onboarding.py [hunk 1]
```diff
@@ -422,3 +423,29 @@ def record_issue_tracker_used(plugin, project, user, **kwargs):
...
         issue_tracker=plugin.slug,
     )
+
+
+@integration_added.connect(weak=False)
+def record_integration_added(integration, organization, user, **kwargs):
+    task = OrganizationOnboardingTask.objects.filter(
+        organization_id=organization.id,
+        task=OnboardingTask.INTEGRATIONS,
+    ).first()
```
Comment (scefali) [inline src/sentry/receivers/onboarding.py:433]: @Neo-Zhixing This is a great case where you can use `get_or_create` to reduce some of the duplication

Comment (Neo-Zhixing) [reply src/sentry/receivers/onboarding.py:433]: Nope, can't do that.
When we create, we always create the same item.
When we update, the updated item relies on the previous state.
These are two distinct behaviors that are not covered by get or create.

Comment (scefali) [reply src/sentry/receivers/onboarding.py:433]: @Neo-Zhixing can you use an F expression? https://docs.djangoproject.com/en/4.0/ref/models/expressions/#f-expressions

Comment (Neo-Zhixing) [reply src/sentry/receivers/onboarding.py:433]: No - we would've been able to do that if this is a jsonb column. But it's saved as text in the database so PostgreSQL wouldn't know what to do with it

**Review by scefali — APPROVED** [review]

**Pull request merged**